exporter/clickhouseexporter/integration_test.go

```diff
@@ -46,36 +51,78 @@ func getContainer(req testcontainers.ContainerRequest) (testcontainers.Container
 	return container, nil
 }
 
-func createClickhouseContainer(image string) (testcontainers.Container, string, string, error) {
+type clickhouseEnv struct {
+	NativeEndpoint       string
+	HTTPEndpoint         string
+	SecureNativeEndpoint string
+	HTTPSEndpoint        string
+}
+
+type stdoutLogConsumer struct{}
+
+func (*stdoutLogConsumer) Accept(l testcontainers.Log) {
+	fmt.Print(string(l.Content))
+}
+
+func createClickhouseContainer(image string) (testcontainers.Container, *clickhouseEnv, error) {
 	port := randPort()
 	httpPort := port + 1
+	tlsPort := port + 2
+	httpsPort := port + 3
+
+	_, b, _, _ := runtime.Caller(0)
+	basePath := filepath.Dir(b)
+
+	fileMount := func(testDataPath, containerPath string) testcontainers.ContainerFile {
+		return testcontainers.ContainerFile{
+			HostFilePath:      path.Join(basePath, "testdata", testDataPath),
```

Comment:
Not a blocking comment, but I've had some success with using an `embedFs` rather than trying to extract the working directory via the caller info at runtime.

I've found it's slightly cleaner to reason about, and handles refactoring easier in the future.